### PR TITLE
Switch to TTF fonts for Russian build

### PR DESCRIPTION
**Pull request by DarthGandalf**
The fonts themselves will be chosen later, this is a temporary solution
to workaround limitation of TeX fonts

### CODE REVIEW

**Comment by qwrtln** [comment]
Looks good. Will approve after the linter is merged.

**Comment by DarthGandalf** [comment]
Actually it looks bad :)

Will need to find some better font at least for headers. The text looks more or less alright

This is good enough for now to fix the capitalization issues you saw

**Comment by qwrtln** [comment]
Yeah, I mean the code, not the font haha.

**Review by qwrtln — APPROVED** [review]

**Review by Heegu-sama — APPROVED** [review]

**Pull request merged**
